### PR TITLE
[WEB-615] Cleanup mess with tests and env vars

### PR DESCRIPTION
**Pull request by gh3r**

### CODE REVIEW

**Comment by gh3r** [comment]
Trying also to unlock firefox tests but currently, selenium is not detecting geckodriver.
If it won't work still I will drop support for it.

**Comment by gh3r** [comment]
Looks like selenium works with geckodriver now. But there are still some issues with tests, they are really slow on firefox, will research it later in other task (WEB-617).

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 gatsbyjs
+Copyright (c) 2015 Syncano
```
Comment (tomsonpl) [inline LICENSE]: why not 2017?

File: README.md [hunk 2]
```diff
@@ -43,9 +43,7 @@ are copied to `public` folder.
 # E2E Testing
 
 We are using [Nightwatch.js](http://nightwatchjs.org/) for e2e testing.
-**Nightwatch.js** requires few binary files which can be installed via proper NPM command:
-
-    $ npm run e2e-setup
+**Nightwatch.js** requires few binary files which can be installed but they will be installed when tests start.
```
Comment (tomsonpl) [inline README.md]: a few

**Comment by adamwardecki** [comment]
lgtm

**Pull request merged**